app/assets/javascripts/discourse/app/components/header/icons.gjs

```diff
@@ -27,9 +27,25 @@ export function clearExtraHeaderIcons() {
 export default class Icons extends Component {
   @service site;
   @service currentUser;
+  @service siteSettings;
   @service header;
   @service search;
 
+  get showHamburger() {
+    // NOTE: In this scenario, we are forcing the sidebar on admin users,
```

Comment:
This weird combo is used in a few places, might want to move this condition into sidebar state